src/bg_triangles.c

```diff
@@ -1,3 +1,4 @@
+#include <stdlib.h> // abs()
```

Comment:
Is this really needed for abs? Cos the `abs` optimisation is automatically handled by agbcc, I guess just for the types?

Reply:
Yeah, agbcc just doesn't give any warnings on it